windows-10-setup.md

```diff
@@ -108,6 +108,7 @@ Install the following VS Code extensions
 - Prettier
 - Remote - WSL
 - Live Share
+- Handlebars (the one by André Junges)
```

Comment:
Could you add this extension in the output for step 3.4, as well? I suspect it will be
`andrejunges.Handlebars`

Reply:
I wish I could say that I had considered this already, but that wouldn't be the truth!

However when I run `code --list-extensions`, it doesn't list this Handlebars extension. I thiiiiink it only lists extensions installed into WSL-Ubuntu, whereas the Handlebars extension installs locally.

Reply:
Hm! Okie doke, then. Thanks for checking. 😄